### DISCUSSION TITLE
Discussion: SUNNY to buyout 15% of Cashio DAO

### DISCUSSION THREAD

**Discussion by ghost** [SunnyAggregator/governance#5] [General]
Hello, JD here. I'm leading development at [Cashio Protocol](https://cashio.app) alongside 0xGhostChain. We have a contract similar in functionality to abracadabra, but currently only takes one type of collateral, which is staked in Quarry & Sunny via Crate & Arrow protocol in order to create a treasury that generates lossless profit as long as CASH (our stablecoin) is in circulation.

The next phase of our project involves the creation of a DAO to leverage the funds farmed (SBR/SUNNY) to participate in the Saber Wars by allowing our governance token holders to vote on how the treasury's permalocked SBR/SUNNY votes on increasing Cash-paried LP emissions. We hope to be the neutral pair stablecoin on Saber with the deepest liquidity, and hope to attract liquidity by offering utility by voting on permalocked veSUNNY/SBR pools to increase Cash-pared LP yields. You can read more [in this medium article](https://medium.com/@cashioapp/introducing-the-cashio-dao-363e6a51c7ad) and if you want to deep dive further explore [our new docs](https://docs.cashio.app/).

Currently, our treasury has ~$400k in SBR/SUNNY generated from only using USDT-USDC LPs as collateral (very, very minimal yield). In the future, the DAO will vote to add other stable pairs (hopefully more decentralized ones) as collateral which generates more yield from the CASH issued. Regardless, our goal here is to bootstrap our 'proxy gauges' with $1M in SBR and $1M in SUNNY, each of which will in turn allocate 15% of our governance token to veSBR/veSUNNY holders in the form of [SAVE tokens](https://docs.tribeca.so/features/save-tokens).

We want to give SBR/SUNNY locked voters to be able to have a big say in how our DAO develops. and we think that by using SAVE tokens - and [permalocking](https://docs.tribeca.so/features/permalock%20copy) the tokens we receive, we can align both parties without fear of dumping on either side. $1M for 15% governance can also be considered discounted when you take into account our TVL ~$28m, and that 15% will be ~1/3 of the circulating supply at launch. 

Proposed Distribution:
* **Early Adopter (17% of FDMC)**

   * All goes to veCOW holders over the course of 1 year
   * Token distributed is CGT-SAVE-1YR which converts to a 1yr-locked veCGT
   * Increased allocation **significantly** to provide value to veCOW holders for their early support, and to make up for the lack of communication/explanation that went into the COW DAO. 

* **SBR Buyout (15% of FDMC)**

  * All goes to veSBR holders over the course of 1 year
  * Token distributed is CGT-SAVE-1YR

* **SUNNY Buyout (15% of FDMC)**

  * All goes to veSUNNY holders over the course of 1 year
  * Token distributed is CGT-SAVE-1YR

* **Team + Advisors (15% of FDMC)**
   * Increased allocation to account for key advisors, and to give the developers a strong voice in the development of the platform. 
* **DAO (38%)**
   * Significantly decreased allocation to limit unnecessary dilution to other stake holders. 
  * Not too large that holders will have a clear incentive to burn, but not too small so that the DAO has no governance supply for strategic initiatives.
      * Examples include: Strategic sales (socean, raydium), future buyouts, emissions for single-sided CASH staking, supply for a burn-based NFT (similar to Marinade), other BD incentives, etc.
      
      
      **Benefits to veSUNNY holders**
      * Additional incentive to lock sunny for veSUNNY
      * Access to governance token with "layer 2" governance power over SUNNY. 
      * Access to one of the only tokens on Solana with high circulating supply at launch compared to the FDMC. 
      * Access to higher yields on Cash Paired pools which can be decided by veSUNNY holders.
      
     We had a similar discussion in the SBR forum, with a lot of information that was put together to create the updated allocation proposed above. You can find that discussion in the following link: https://gov.saber.so/t/discussion-sbr-to-buyout-15-of-cashio-dao/197/8
     
     Would love to hear any feedback on the edits we made to tokenomics, and discuss anything that's unclear about what cashio is/is going to be when we become a DAO.

**Comment by d0bby5** [comment]
Lots of discussion on this proposal in the Saber governance forum. Would highly recommend that anyone seeing this for the first time read up on it [here](https://gov.saber.so/t/discussion-sbr-to-buyout-15-of-cashio-dao/197). 

Looking forward to seeing any feedback from @SuryaKhosla on this idea. Seems to be garnering a lot of attention in the Saber community.

**Comment by tmnxeq** [comment]
Thanks for the proposal @jay-dx. 

Can you clarify if Team + Advisors token are also distributed as CGT-SAVE-1YR token?

**Comment by autocatalytic** [comment]
FROM DISCORD: hushedfail#3410 — Today at 3:35 AM

Hi @George Washington ! Thanks a lot for the proposal! I'm commenting here because I don't have an anon github account (sunny: I don't really like the gov forum being on github tbh)

I have a few questions:

- The team tokens are not specified in what form. Also CGT-SAVE-1YR?
- $1M in sunny tokens is really a lot of sunny. Almost 10% of the current market cap. It's also way more in terms of market cap then in the SBR case which has a market cap of ~3X that of sunny, so there it's ~3-4% or so (not exact figures). For one protocol to immediately get 10% of another protocol's votes sounds like a lot, maybe too much. I think this is a product of the current low price and the super thin liquidity. To be able to "buy" $1M of sunny at a price of $0.004 doesn't feel "fair" for me tbh. However a discounted rate based on a market buy is also not fair, as there isn't even $1M in liquidity (at least on orca).

So how should a good price for the sunny tokens be determined? 
I mean if the current price at date X would be determined, anyone with a "small" bag of 50K or so can just pump the price up just before and you'd be left with way less tokens as well
So determining a price for sunny that's not market based might also be in your advantage
